en/docs/learn/extended-self-registration-api-and-account-confirmation-api.md

```diff
@@ -0,0 +1,473 @@
+    channel and the user has not specified a preferred channel, the notification will be sent via 
+    the channel that is given in the request as a claim value. 
+        
+        **Example**: If only the mobile number has been provided, mobile number is considered as 
+        the preferred channel.  
+    
+    2. If the user has specified a preferred channel and the preferred channel matches with the 
+    given claim values, the confirmation info will be sent via the user specified preferred channel.
+    
+        **Example**: If the preferred channel is specified as SMS and the provided 
+        claim value is a mobile number, an OTP will be sent to the user.
+        
+    3. If the user has specified a preferred channel and the preferred channel does not match with 
+    the given claim values, a **400 ERROR** is returned. 
+    
+        **Example**: The preferred channel is specified as SMS but there is no value for the 
+        mobile number.
+        
+        !!! Note
+            Each notification channel is bound with the claim channel.
+            <table>
+                <tr>
+                    <th>Notification Channel</th>
+                    <th>Bounded Value Claim</th>
+                </tr>
+                <tr>
+                    <td>EMAIL</td>
+                    <td>http://wso2.org/claims/emailaddress</td>
+                </tr>
+                <tr>
+                    <td>SMS</td>
+                    <td>http://wso2.org/claims/mobile</td>
+                </tr>
+            </table>
+            
+            !!! warning
+                Changing the bounded value claim will cause errors.
+            
+     4. If the user has provided both email and a mobile as communication channels and has also specified 
+     a preferred channel, the notification will be sent via the preferred channel.
+     
+     5. If the user has provided both email and mobile as communication channels but has not 
+     specified a preferred channel, notification will be sent via the server default channel. 
+     
+4. Once the communication channel is resolved, an event is triggered. Once the event is 
+triggered, the notification handlers will send notifications to the user.
+   
+    The event names will be in the following format.
+    
+    <table>
+    <tr>
+        <th>Channel</th>
+        <th>Event Name</th>
+    </tr>
+    <tr>
+        <td>SMS</td>
+        <td>TRIGGER_SMS_NOTIFICATION</td>
+    </tr>
+    <tr>
+        <td>Email</td>
+        <td>TRIGGER_NOTIFICATION</td>
+    </tr>
+    </table>     
+    
+    !!! note
+        If you have any custom notification handlers, you need to subscribe the notification handler 
+        to above events.
+
+### **API Requests and Responses**
+
+There are three scenarios associated with user self registration.
+
+1. **Self registration with notifications internally managed and the account is locked on creation.**
```

Comment:
Shall we change this to 'Self registration with internally managed notifications and account lock on creation'?

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549